src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -168,7 +174,12 @@ void CoreWorkerDirectActorTaskSubmitter::DisconnectActor(const ActorID &actor_id
   RAY_LOG(DEBUG) << "Disconnecting from actor " << actor_id;
   absl::MutexLock lock(&mu_);
   auto queue = client_queues_.find(actor_id);
-  RAY_CHECK(queue != client_queues_.end());
+  if (queue == client_queues_.end()) {
```

Comment:
Why do we need to update this line? I don't see usages of `client_queues_.erase(...)`.